src/transformers/__init__.py

```diff
@@ -438,6 +442,25 @@
         name for name in dir(dummy_timm_objects) if not name.startswith("_")
     ]
 
+# Detectron2-backed objects
+if is_detectron2_available():
+    _import_structure["models.layoutlmv2"].extend(
+        [
+            "LAYOUTLM_V2_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "LayoutLMv2ForQuestionAnswering",
+            "LayoutLMv2ForSequenceClassification",
+            "LayoutLMv2ForTokenClassification",
+            "LayoutLMv2Model",
+            "LayoutLMv2PreTrainedModel",
+        ]
+    )
```

Comment:
I don't think we need to create a new dummy file just for LayoutLMv2. Like for Tapas that requires torch scatter as an extra dep, I think we should just make sure detectron2 is installed when initializing the model.

Reply:
Ok, waiting for @LysandreJik to confirm this.

Reply:
As with all dummy objects related questions, I agree with @sgugger's judgment!

Reply:
Ok, will update so that it works similar to TAPAS.